crates/phactory/src/lib.rs

```diff
@@ -398,6 +402,10 @@ impl<Platform: pal::Platform> Phactory<Platform> {
         Ok(data)
     }
 
+    fn persistent_runtime_data(&self) -> Result<PersistentRuntimeData, Error> {
```

Comment:
Maybe add a comment like: /// Loads the persistent runtime data from the sealing path

Reply:
OK